### PR TITLE
Issue/1486: Create labels for all moons

### PR DESCRIPTION
**Pull request by Arohdin**
`2bb7252cec69b1014ad3cbd15f00cd7cdd36b19b` : Updated fade behavior and label sizes for all planets (+Pluto) in the solar system. Before they had very different sizes and different fade behaviors.
Should be more consistent and visually pleasing now.

 `84d93d6f3a97e9f0c185beb8c5f8315d6caa4956` : Added labels for all major (and minor) moons in the solar system.
 Added specific `tags` for major and minor moons so that they can be easily set up with keybindings/actions if needed.
 
_( `508a1a9e1bdde4e53e9fd9d927dbd7990bdea7cc` : Added one tag that I missed adding 😒)_

Example video:

https://user-images.githubusercontent.com/9076357/215102940-153d65b6-ec6a-44bb-943f-f90584f3956e.mp4

### CODE REVIEW

File: data/assets/scene/solarsystem/dwarf_planets/pluto/charon/charon.asset [hunk 1]
```diff
@@ -50,11 +50,38 @@ local Charon = {
...
+    EnableFading = true,
+    FadeUnit = "Mm",
+    FadeDistances = { 250.0, 500.0 },
+    FadeWidths = { 150.0, 250.0 }
+  },
+  Tag = { "solarsystem_labels", "moon_labels", "major_moon_labels" },
+  GUI = {
+    Name = "Cheron Label",
+    Path = "/Solar System/Dwarf Planets/Pluto/Moons",
+    Description = "Label for Pluto's moon Cheron"
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/pluto/charon/charon.asset:74]: -> Charon

File: data/assets/scene/solarsystem/planets/jupiter/jupiter.asset [hunk 2]
```diff
@@ -48,10 +48,14 @@ local JupiterLabel = {
     Type = "RenderableLabel",
     Text = "Jupiter",
     FontSize = 70.0,
-    Size = 8.77,
-    MinMaxSize = { 1, 100 },
+    Size = 8.75,
+    MinMaxSize = { 1, 50},
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/jupiter/jupiter.asset:52]: I like to have a space between the number and the opening and closing `{`/`}`

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset [hunk 3]
```diff
@@ -207,22 +207,61 @@ local anankeGroup = {
   }
 }
 
+--Generate all moon labels
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset:210]: I'd usually have a space between the `--` and the comment to make it easier to read, but that's personal preference

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset [hunk 4]
```diff
@@ -207,22 +207,61 @@ local anankeGroup = {
...
+      EnableFading = true,
+      FadeUnit = "Gm",
+      FadeDistances = { 1.5, 90.0 },
+      FadeWidths = { 0.5, 90.0 }
+    },
+    Tag = { "solarsystem_labels", "moon_labels", "minor_moon_labels" },
+    GUI = {
+      Name = moon.Identifier .. " Label",
+      Path = "/Solar System/Planets/Jupiter/Moons",
+      Description = "Label for Jupiter's moon " .. moon.Identifier .. " (Ananke group)"
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset:235]: Should this be `moon.GUI.Name` instead?  In most cases those would be the same, but they don't have to be

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset [hunk 5]
```diff
@@ -207,22 +207,61 @@ local anankeGroup = {
...
+
+
 local nodes = proceduralGlobes.createGlobes(anankeGroup)
 
 asset.onInitialize(function()
   for i, node in ipairs(nodes) do
     openspace.addSceneGraphNode(node)
   end
+
+  for i, moonlabel in ipairs(moon_labels) do
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset:248]: The Lua-esque way of dealing with the parameter you don't want is to use `_` instead [which should have happened in the list above as well], so:
```lua
for _, moonLabel in ipairs(moon_labels) do
```

File: data/assets/scene/solarsystem/planets/jupiter/minor/carpo_group.asset [hunk 6]
```diff
@@ -35,15 +35,80 @@ local carpoGroup = {
...
+    },
+    Tag = { "solarsystem_labels", "moon_labels", "minor_moon_labels" },
+    GUI = {
+      Name = moon.Identifier .. " Label",
+      Path = "/Solar System/Planets/Jupiter/Moons",
+      Description = "Label for Jupiter's moon " .. moon.Identifier .. " (Carpo group)"
+    }
+  }
+
+  -- Hacky solution to issue where the main label gets culled due to it's onscreen size.
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/jupiter/minor/carpo_group.asset:67]: -> its

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset [hunk 7]
```diff
@@ -207,22 +207,61 @@ local anankeGroup = {
...
+local moon_labels = {}
+
+for i, moon in ipairs(anankeGroup) do
+  moon_labels[i] = {
+    Identifier = moon.Identifier .. "Label",
+    Parent = moon.Identifier,
+    Renderable = {
+      Enabled = false,
+      Type = "RenderableLabel",
+      Text = moon.Identifier,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset:220]: I think this should be `moon.GUI.Name` instead

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset (hunk 4 above)
Comment (alexanderbock) [reply data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset:235]: Same for a bunch of other places

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset (hunk 5 above)
Comment (alexanderbock) [reply data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset:248]: Same for a bunch of other places

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset (hunk 4 above)
Comment (Arohdin) [reply data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset:235]: not all moons have GUI name set it seems (see image below).
Should I update all assets that are missing this property?
Or should I do a conditional that tries to get moon.GUI.name but have fallback for Identifier?

moon_gui

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset (hunk 7 above)
Comment (Arohdin) [reply data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset:220]: See same comment as for: [data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset](https://github.com/OpenSpace/OpenSpace/pull/2453/files/508a1a9e1bdde4e53e9fd9d927dbd7990bdea7cc#diff-59c1a7d7c5055671ede6b9b0f84e990f8ed5bb46bf54241ded3b73e3f9f43de8)

File: data/assets/scene/solarsystem/planets/jupiter/jupiter.asset (hunk 2 above)
Comment (Arohdin) [reply data/assets/scene/solarsystem/planets/jupiter/jupiter.asset]: Fixed in: [055eab06540015eba3138c73cb4cb4bf6f36b5ad](https://github.com/OpenSpace/OpenSpace/commit/055eab06540015eba3138c73cb4cb4bf6f36b5ad)

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset (hunk 3 above)
Comment (Arohdin) [reply data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset]: Fixed in: [055eab06540015eba3138c73cb4cb4bf6f36b5ad](https://github.com/OpenSpace/OpenSpace/commit/055eab06540015eba3138c73cb4cb4bf6f36b5ad)

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset (hunk 5 above)
Comment (Arohdin) [reply data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset]: Fixed in: [055eab06540015eba3138c73cb4cb4bf6f36b5ad](https://github.com/OpenSpace/OpenSpace/commit/055eab06540015eba3138c73cb4cb4bf6f36b5ad)

File: data/assets/scene/solarsystem/planets/jupiter/minor/carpo_group.asset (hunk 6 above)
Comment (Arohdin) [reply data/assets/scene/solarsystem/planets/jupiter/minor/carpo_group.asset]: Fixed in: [055eab06540015eba3138c73cb4cb4bf6f36b5ad](https://github.com/OpenSpace/OpenSpace/commit/055eab06540015eba3138c73cb4cb4bf6f36b5ad)

File: data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset (hunk 4 above)
Comment (Arohdin) [reply data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset:235]: Note:
It was decided that conditional will be implemented.

**Pull request merged**